config.json

```
@@ -487,6 +487,27 @@
                 "math"
             ]
         },
+        {
+            "uuid": "a534b8ed-da88-4945-bb25-f6633977740b",
+            "slug": "pythagorean-triplet",
+            "core": true,
```

```suggestion
            "core": false,
```

For now, the core track is full. Let's leave it as floating exercise until we know where it would work best.